exporter/clickhouselogsexporter/migrations/000006_tag_attributes.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS signoz_logs.tag_attributes ON CLUSTER cluster (
+    timestamp DateTime CODEC(ZSTD(1)), 
+    tagKey LowCardinality(String) CODEC(ZSTD(1)),
+    tagType LowCardinality(String) CODEC(ZSTD(1)),
+    tagDataType LowCardinality(String) CODEC(ZSTD(1)),
+    stringTagValue String CODEC(ZSTD(1)),
+    numberTagValue Nullable(Float64) CODEC(ZSTD(1))
```

Comment:
How are booleans handled?

Reply:
Since the boolean value is either true/false, the API returns those two values. We are avoiding creating a new column for that. i.e the entry will be present but there won't be any value column for that.